lib/index.ts

```diff
@@ -95,18 +146,28 @@ export const convertCircuitJsonToLbrn = (
     connMap,
     topNetGeoms: new Map(),
     bottomNetGeoms: new Map(),
+    topMarginNetGeoms: new Map(),
+    bottomMarginNetGeoms: new Map(),
     origin,
-    includeCopper: options.includeCopper ?? true,
-    includeSoldermask: options.includeSoldermask ?? false,
-    soldermaskMargin: options.soldermaskMargin ?? 0,
+    includeCopper,
+    includeSoldermask,
+    soldermaskMargin,
     includeLayers,
+    traceMargin,
+    laserSpotSize,
+    topTraceMarginCutSetting,
+    bottomTraceMarginCutSetting,
   }
 
+  // Initialize net geometry maps
   for (const net of Object.keys(connMap.netMap)) {
     ctx.topNetGeoms.set(net, [])
     ctx.bottomNetGeoms.set(net, [])
+    ctx.topMarginNetGeoms.set(net, [])
+    ctx.bottomMarginNetGeoms.set(net, [])
```

Comment:
Really bad naming. What are these? Are these all copper? Why is margin copper different?

one should be “topFillNetGeoms” and one should be “topOutlineNetGeoms”

if we merged this as is the codebase would be unrecoverable! This is very unclear- took me a while to understand what was happening